tests/baselines/reference/computedPropertyNamesWithTypeOfOperator.errors.txt

```diff
@@ -0,0 +1,26 @@
+tests/cases/compiler/computedPropertyNamesWithTypeOfOperator.ts(11,34): error TS18036: Computed property names can only be named with well-known symbols without the 'typeof' operator. Try replacing 'typeof [object Object]' with '[object Object]'.
+tests/cases/compiler/computedPropertyNamesWithTypeOfOperator.ts(11,42): error TS2585: 'Symbol' only refers to a type, but is being used as a value here. Do you need to change your target library? Try changing the `lib` compiler option to es2015 or later.
+tests/cases/compiler/computedPropertyNamesWithTypeOfOperator.ts(13,35): error TS2585: 'Symbol' only refers to a type, but is being used as a value here. Do you need to change your target library? Try changing the `lib` compiler option to es2015 or later.
+
+
+==== tests/cases/compiler/computedPropertyNamesWithTypeOfOperator.ts (3 errors) ====
+    interface SymbolConstructor {
+      observable: symbol;
+    }
+    
+    type Observer<T> = (x: T) => void;
+    
+    interface Observable<T> {
+      subscribe(observer: Observer<T>): { unsubscribe(): void }
+    }
+    
+    declare function from1<T>(obj: { [typeof Symbol.observable](): Observable<T> }): Observable<T>;
+                                     ~~~~~~~~~~~~~~~~~~~~~~~~~~
+!!! error TS18036: Computed property names can only be named with well-known symbols without the 'typeof' operator. Try replacing 'typeof [object Object]' with '[object Object]'.
+                                             ~~~~~~
+!!! error TS2585: 'Symbol' only refers to a type, but is being used as a value here. Do you need to change your target library? Try changing the `lib` compiler option to es2015 or later.

```

Comment:
I'm also not sure where this error is coming from. When I run type inference on this file outside of the tests, it doesn't pop up 🤔

Reply:
You'll need to include es2015.symbol as part of the `lib`. `SymbolConstructor` isn't used in the base lib.

In tests, you can do this with `// @lib: es5, es2015.symbol` if I recall correctly.